gui/src/main.rs

```diff
@@ -98,7 +98,10 @@ impl Application for GUI {
     }
 
     fn new(config: Config) -> (GUI, Command<Self::Message>) {
-        let logger = Logger::setup(LevelFilter::INFO);
+        let log_level = std::env::var("LOG_LEVEL")
+            .ok()
+            .and_then(|s| LevelFilter::from_str(&s).ok());
```

Comment:
Next time let's not ignore errors, we can print it or something but silently ignoring it is maddening when debugging.